<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_disp_entrada_ingresar</class>
 <widget class="QMainWindow" name="vtn_disp_entrada_ingresar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingreso de Dispositivo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_disp_entrada">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>22</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingreso Dispositivo de Entrada</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tipo_dispositivo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Escoja el Dispositivo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_marca">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Marca:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tipo_entrada">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de Entrada</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tipo_dispositivo">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Rat√≥n</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Teclado</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txt_marca">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tipo_entrada">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_guardar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
 </widget>
 <resources/>
 <connections/>
</ui>
